src/averageAges.js

```diff
@@ -37,7 +42,18 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  // eslint-disable-next-line max-len
+  const isMother = people.filter(person => person.mother).map(person => person.mother);
+  const woman = withChildren
+    ? people.filter(person => person.sex === 'f'
+    && isMother.includes(person.name))
```

Comment:
instead of filter with includes you can check if woman is mother via `some` method